requirements/requirements-dev.txt

```diff
@@ -1,2 +1,3 @@
 click>=7.0
 ruamel.yaml>=0.16.12
+pre-commit==2.17.0
```

Comment:
I assume that the pre-commit dependencies are documented as part of the pre-commit config?

Reply:
Yes. pre-commit manages its own virtual environments so they are entirely independent from other dependencies in the user's environment. See https://github.com/glotzerlab/signac-flow/blob/master/.pre-commit-config.yaml